### DISCUSSION TITLE
Open Disclosure About Premium Licensing Feature(s) In Documentation

### DISCUSSION THREAD

**Discussion by Sn3akyP3t3** [dani-garcia/vaultwarden#2522] [General]
It didn't take me extremely long to discover that I didn't need to pay for licensing to BitWarden to enable family shared capabilities, but it did cause quite a bit of confusion on my end for at least a week.  I even reached out to BitWarden for support not knowing that this container was not an officially supported build.  I didn't mean to do that and I'm sure I confused them as well.

What I'm asking for here as actionable from this is clarity that these premium features are enabled out of box and don't require additional licensing.  Whether or not hints that a person should drop payment for upstream support is worthy to note, but not absolutely necessary as part of my request.  If this was declared in the Wiki or documentation of setup I would likely have noticed it and not proceeded to try to apply a license.  This is my first time working with the BitWarden application so I'm coming from zero familiarity of things and official naming.

**Comment by BlackDex** [comment]
Well, you probably didn't read the README, wiki, the startup of the container, the docker hub README or every other location here.
On both the main page of this repo as on the main page on docker hub of the container you see this message.
![image](https://user-images.githubusercontent.com/610450/171226205-8669a01e-604f-4c6e-a7e9-80433ea3a698.png)

You also see a message when starting the container which a similar message.
So, i think we have placed it at a lot of locations. Hell, we even modified the `Get Help` link within the web-vault points this this repo.

At the wiki we also state this project is an **unofficial Bitwarden server implementation**.
And below that there is a list of features this project supports.

So, if you have any more suggestions what we need to do besides that to make it more clear, please let us know.
